Backend/app/models/query-models/base-query-data.model.js

```diff
@@ -18,7 +18,7 @@ const QueryData = function() {
   }
 
   this.appendUpdateParam = function(param, column) {
-    if (!param) return;
+    if (param === null || param === undefined) return;
```

Comment:
You can do param == null (both undefined and null will be caught but not other falsy values like 0, [], {}, etc.)